static/src/javascripts/projects/common/utils/raven.js

```diff
@@ -46,8 +46,11 @@ define([
                     }
                 }
 
-                return config.switches.enableSentryReporting &&
+                return adBlockerOn.then(function(adBlockerOn){
```

Comment:
Perhaps we could keep it sync using the same technique than in `tech-feedback`; I think it is safe to assume the promise will be resolved before this is even executed most of the times.